### PR TITLE
fix(iati-etl): upload transformed data at once - noissue

### PR DESCRIPTION
**Pull request by kalinchernev**
# PR description

Example [environment](http://eubfr-chernka00-demo-dashboard-client-iati.s3-website.eu-central-1.amazonaws.com/#/files)

You can trace `chernka00` in s3 raw and harmonized buckets.

## QA Checklist

When you add a new ETL/producer, please check for the following:

* [ ] Producer's secrets are stored safely
* [ ] Ensure the ETL is added to the corresponding `scripts/` for automated deployment and deletion 
* [ ] There is at least 1 unit test with a jest snapshot for the transform function of the ETL
* [ ] Update the file PRODUCERS_DATA_AVAILABILITY_GRID.md indicating which fields are available in source data of ETL
* [ ] Ensure there is (flow/jsdocs) documentation in the `tranform.js` file which is to be used for automated documentation
* [ ] Generate the necessary documentation pages for the new ETL by `yarn docs:md`

### CODE REVIEW

File: services/ingestion/etl/iati/csv/src/events/onParseCSV.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ import { STATUS } from '@eubfr/logger-messenger/src/lib/status';
 // Import logic
 import { extractMessage } from '../lib/sns';
 import transformRecord from '../lib/transform';
```
Comment (yhuard) [inline services/ingestion/etl/iati/csv/src/events/onParseCSV.js]: you can delete the `../lib/uploadFromStream.js` file

File: services/ingestion/etl/iati/csv/src/events/onParseCSV.js [hunk 2]
```diff
@@ -53,7 +52,7 @@ export const handler = async (event, context, callback) => {
     (record, cb) => {
       try {
         const data = transformRecord(record);
-        return cb(null, `${JSON.stringify(data)}\n`);
+        return cb(null, `${JSON.stringify(data)}\r\n`);
```
Comment (yhuard) [inline services/ingestion/etl/iati/csv/src/events/onParseCSV.js]: why? it was working fine until now, wasn't it?

**Review by yhuard — COMMENTED** [review]
only questions

File: services/ingestion/etl/iati/csv/src/events/onParseCSV.js (hunk 2 above)
Comment (kalinchernev) [reply services/ingestion/etl/iati/csv/src/events/onParseCSV.js]: I'll remove this change

**Review by yhuard — APPROVED** [review]
The code looks okay to me

**Pull request merged**